examples/window/viewports.rs

```diff
@@ -0,0 +1,218 @@
+use bevy::{
+    math::{clamp, Rect},
+    prelude::*,
+    render::{
+        camera::{ActiveCameras, Camera},
+        render_graph::{base::MainPass, CameraNode, PassNode, RenderGraph},
+        surface::{SideLocation, Viewport, ViewportDescriptor},
+    },
+};
+
+/// This example creates a second window and draws a mesh from two different cameras.
+fn main() {
+    App::build()
+        .insert_resource(Msaa { samples: 4 })
+        .init_resource::<ViewportLayout>()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup.system())
+        .add_system(viewport_layout_system.system())
+        .run();
+}
+
+fn setup(
+    commands: &mut Commands,
+    mut active_cameras: ResMut<ActiveCameras>,
+    mut render_graph: ResMut<RenderGraph>,
+    asset_server: Res<AssetServer>,
+) {
+    // add new camera nodes for the secondary viewports
+    render_graph.add_system_node("front_view_camera", CameraNode::new("FrontView"));
+    render_graph.add_system_node("side_view_camera", CameraNode::new("SideView"));
```

Comment:
could those strings be stored in consts to promote this usage, and also be more explicit in the difference between the two strings by giving them name?
```rust
const FRONT_CAMERA: &str = "FrontView";
const FRONT_CAMERA_NODE: &str = "front_view_camera";
const SIDE_CAMERA: &str = "SideView";
const FRONT_CAMERA_NODE: &str = "side_view_camera";
```

Reply:
Done, good catch! I've also changed the main pass node name to `base::node::MAIN_PASS` for the same effect.